images/windows/hub/Dockerfile

```diff
@@ -1,7 +1,6 @@
 ARG baseImage="unityci/base:windows-latest"
 
 FROM $baseImage
-ARG hubVersion="3.0.0"
 
 # Install unity hub
-RUN choco install unity-hub --version=%hubVersion --no-progress -y
+RUN choco install unity-hub --no-progress -y
```

Comment:
Why remove the version flag? Is the argument no longer valid?

Reply:
It actually wasn't relevant from the start. Reading more into the readme for the chocolatey package, Unity doesn't store an archive of the older Unity hubs so it will always pull the latest version. The chocolatey script used a hard coded hash for each version to verify the download so putting an older version can cause errors during installs.

Reply:
They must have changed that. I wouldn't have added this param without it being documented.
Nice catch!